tests/testthat/setup.R

```diff
@@ -0,0 +1,9 @@
+library(dplyr)
+
+## listings var labels don't get mucked up by topleft machinery #262
```

Comment:
This comment doesn't belong in setup.R, please relocate it to whever the actual test case was moved to.